src/Resources/contao/classes/DataContainer.php

```diff
@@ -902,8 +902,9 @@ protected function initPicker()
 
 		$this->setPickerValue();
 
-		$strDriver = 'DC_' . $GLOBALS['TL_DCA'][$this->strPickerTable]['config']['dataContainer'];
-		$objDca = new $strDriver($this->strPickerTable);
+		// Create a DataContainer object (see #886)
+		$objDca = new \DataContainer();
```

Comment:
Also, please remember that you solved the **exact same case** in Haste with a `strClass` object (see https://github.com/codefog/contao-haste/pull/75). This should illustrate that we only need a helper object.